### PR TITLE
add status to summary comment of ConsumeResultFields

### PR DESCRIPTION
**Pull request by msallin**

### CODE REVIEW

**Comment by msallin** [comment]
https://github.com/confluentinc/confluent-kafka-dotnet/issues/1537

**Comment by msallin** [comment]
[clabot:check]

File: src/ConfigGen/Program.cs [hunk 1]
```diff
@@ -470,7 +470,7 @@ static string createConsumerSpecific()
         ///     <see cref=""Confluent.Kafka.Consumer{TKey,TValue}.Consume(System.TimeSpan)"" />
         ///     method. Disabling fields that you do not require will improve
         ///     throughput and reduce memory consumption. Allowed values:
-        ///     headers, timestamp, topic, all, none
+        ///     headers, timestamp, topic, status, all, none
```
Comment (edenhill) [inline src/ConfigGen/Program.cs:473]: The message status field is currently only relevant to producer delivery reports.
A consumed message is per definition persisted.

Comment (edenhill) [reply src/ConfigGen/Program.cs:473]: Yeah :+1:

**Pull request merged**